common/interfaces.go

```diff
@@ -0,0 +1,6 @@
+package common
```

Comment:
I think `utils` already serves as this.

Reply:
Yeah, I initially thought as you. But I'm just trying to open a common place to have code that's really common to all. `utils` is kinda "we don't know where to put this" option.

Reply:
It doesn't seem like we use this interface for anything other than cloning interface, so it's not common and there's no guarantee it will be. Furthermore, will there be some kind of cloner which returns a different cloner? 
If anything, `Clone` could have been part of the filter interface with the method `Clone() Filter`.

Reply:
I'm ok in placing it in `utils`. About being Cloners, we have `Policy` and `Policies` besides `Filter`, take a look:

https://github.com/aquasecurity/tracee/pull/3305#discussion_r1411378820